src/index.js

```diff
@@ -206,7 +206,8 @@ function createConfig(options, entry, format) {
 				}),
 				useNodeResolve && nodeResolve({
 					module: true,
-					jsnext: true
+					jsnext: true,
+					browser: options.target === 'browser'
```

Comment:
Can we just avoid the extra option and set `browser: true` always? If you're building for node, you'll use `--external all` anyway, so this will never be hit.

Reply:
I'd just do `browser: options.target!=='node'`.